Neos.Flow/Classes/I18n/Service.php

```diff
@@ -293,7 +293,7 @@ protected function generateAvailableLocalesCollectionByScanningFilesystem()
         $blacklistPattern = $this->getScanBlacklistPattern();
 
         /** @var PackageInterface $activePackage */
-        foreach ($this->packageManager->getAvailablePackages() as $activePackage) {
+        foreach ($this->packageManager->getFlowPackages() as $activePackage) {
             $packageResourcesPath = Files::getNormalizedPath($activePackage->getResourcesPath());
```

Comment:
`getResourcesPath()` is not part of the `PackageInterface` any longer

Reply:
True, the docblock is bad.